app/graphql/types/well_query_type.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Types
+  # The type for Well queries.
+  class WellQueryType < BaseObject
```

Comment:
Does it make sense to add this to a Types::Queries namespace structure?

Reply:
We can if you think it makes it clearer.  I'm just following how GraphQL suggest to lay these classes out.  `BaseObject` is in the `Types` namespace so it might not be appropriate.

Reply:
:+1: I suppose this can always be changed later quite easily if we feel it needs to be